packages/lab/src/date-picker/index.ts

```diff
@@ -2,6 +2,7 @@ export * from "./DatePicker";
 export * from "./DatePickerActions";
 export * from "./DatePickerContext";
 export * from "./DatePickerOverlay";
+export * from "./DatePickerOverlayProvider";
```

Comment:
@mark-tate Does this need to be exported?

Reply:
yes.. if folks want to build their own date picker inputs, they may need

```
  const {
    state: { open },
    helpers: { setOpen },
  } = useDatePickerOverlay();
  ```